OSDOCS-13191#Resizing recovery (#91198)
openshift/openshift-docs: modules/persistent-storage-csi-resizing-procedure.adoc

// Module included in the following assemblies:
//
// * storage/container_storage_interface/persistent-storage-csi-resizing.adoc

:_mod-docs-content-type: PROCEDURE
[id="persistent-storage-csi-resizing-procedure_{context}"]
= Resizing a CSI volume

> 

Reviewer: Consider adding an introductory line or two here.

.Prerequisites
* You are logged in to a running {product-title} cluster.

> * Your persistent volume claim (PVC) is created using a CSI driver that supports volume resizing.

Reviewer: ```suggestion
* Your persistent volume claim (PVC) is created by using a CSI driver that supports volume resizing.
```
Should "is created" be "was created"?

* Your storage back end is configured for dynamic provisioning.

.Procedure

> To resize a volume:

Reviewer: ```suggestion

```
I think you can safely remove this line because it's clear from the title that the procedure is to resize a CSI volume.

> . Ensure that your storage class supports resizing:

Reviewer: ```suggestion
. Ensure that your storage class supports resizing by editing the YAML file for the storage class:
```

+
[source, yaml]
.Sample storage class YAML file
----
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: <sc-name> <1>
provisioner: <provisioner-name> <2>
parameters:
    type: gp2
reclaimPolicy: Delete
allowVolumeExpansion: true <3>
----
<1> Storage class name.
<2> Provisioner name.
<3> To allow resizing, the `allowVolumeExpansion` parameter must be set to `true`.

> . In the desired PVC, increase the `storage` field to trigger resizing:

Reviewer: ```suggestion
. In the YAML file for the PVC that you want to resize, increase the `storage` field to trigger resizing:
```
Feel free to ignore this suggestion if it's incorrect.
